### ISSUE TITLE
coOncoplot improvements for large sample size

### ISSUE THREAD

**Issue by simonlee184** [PoisonAlien/maftools#300]
Hi Anand,
     I tried making a coOncoplot with over 1000 samples, but it either gave a very faint plot, or just white space if I opened it in Adobe Reader, Though if I just used oncoplot to plot the same sample set, it worked fine. Is there a way to fix this issue possibly by removing the whitespace or increasing resolution?

Simon

**Comment by PoisonAlien** [comment]
Hi Simon,
Its always tricky with sample size of this big. One suggestion is to save the plot to a pdf with larger `width` - It should give enough space for the grids to be visible. Meanwhile I will try to make option for custom grid sizes.

**Comment by simonlee184** [comment]
Right, I've done that but even then it didn't work. It was weird in that when I zoomed in and out, I could see the colored boxes clearly for a second and it would disappear. So instead, I opened the file in Illustrator to increase the opacity which helped a bit, but the colors were still faint.
So what was weird about it was that when I plotted both cohorts in oncoplot, the plot looked fine, but when I separated the two in coOncoplot, it gave me a very faint colored plot even with a wider pdf.

Simon
